.github/workflows/onPushToMain.yml

```diff
@@ -0,0 +1,11 @@
+name: version, tag and github release
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  release:
+    uses: salesforcecli/github-workflows/.github/workflows/githubRelease.yml@gbockus/updateReleaseToIncludeTokenInput
```

Comment:
we'll hopefully move this back to the main branch soonish.  I have one pending update to push.